ui/narrative/methods/import_tsv_as_expression_matrix_from_staging/display.yaml

```diff
@@ -21,16 +21,14 @@ suggestions :
 
 parameters :
     staging_file_subdir_path :
-        ui-name : |
-            TSV File Path
+        ui-name : TSV file path
         short-hint : |
             File with expression data in TSV format with Features in rows and Conditions in columns
         long-hint  : |
             The filename should end with ".tsv"
 
     matrix_name :
-        ui-name : |
-            Expression Matrix Object Name
+        ui-name : Expression Matrix object name
```

Comment:
Does matrix need to be capitalized?

Reply:
apparently so if it is an Expression Matrix